apps/webapp/src/app/theme/use-theme.tsx

```diff
@@ -0,0 +1,39 @@
+import { createTheme, Theme } from '@mui/material';
+import { useThemeMode } from './use-theme-mode';
+
+export const useTheme = (): [
+  theme: Theme,
+  themeMode: { change: () => void }
+] => {
+  const [mode, themeModeChange] = useThemeMode();
+  const theme = createTheme(
+    {
+      palette: {
+        mode: localStorage.getItem('theme') === 'light' ? 'light' : 'dark',
+        ...(localStorage.getItem('theme') === 'light'
+          ? {
```

Comment:
na dłuższą metę to się będzie słabo skalowało (taki if theme) ale póki co pewnie jest ok ;)